package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "swagger-jsdoc": "^6.2.8"
+  },
+  "devDependencies": {
+    "@types/swagger-jsdoc": "^6.0.4",
+    "@types/swagger-ui-express": "^4.1.6",
+    "swagger-ui-express": "^5.0.0"
+  }
+}
```

Comment:
You've accidentally installed these in the root of your Project instead of your API
`swagger-ui-express` can be installed as a regular dependency and not a dev dependency

Reply:
Once you delete these files, if git still tries to hold onto them you can use git rm. But that is usually if you accidentally commit a file that was meant to be .gitignored
https://git-scm.com/docs/git-rm